test/uk/gov/hmrc/merchandiseinbaggage/pact/MibConnectorContractSpec.scala

```diff
@@ -33,13 +33,14 @@ import uk.gov.hmrc.merchandiseinbaggage.{BaseSpecWithApplication, CoreTestData}
 import java.io.File
 import java.time.LocalDate
 
-class MibConnectorContractSpec extends BaseSpecWithApplication with CoreTestData with MibConfiguration {
+class MibConnectorContractSpec extends BaseSpecWithApplication with CoreTestData {
 
   implicit val formats: DefaultFormats.type = DefaultFormats
 
   val CONSUMER: String           = "merchandise-in-baggage-frontend"
   val PROVIDER: String           = "merchandise-in-baggage"
   val mibConnector: MibConnector = injector.instanceOf[MibConnector]
+  val config: AppConfig          = injector.instanceOf[AppConfig]
```

Comment:
There is `appConfig` in `BaseSpecWithApplication` but please remove as it is not needed, thanks